src/lib/time.js

```diff
@@ -270,25 +275,42 @@ var $builtinmodule = function (name) {
     %Z  Time zone name (no characters if no time zone exists).   
     %%  A literal '%' character.         
     */
-    // accept2dyear
-    mod.accept2year = function accept2year() {
-        throw new Sk.builtin.NotImplementedError("accept2year is not implemented")
-    };
 
-    // strftime()
-    mod.strftime = function strftime() {
-        throw new Sk.builtin.NotImplementedError("strftime is not implemented")
-    };
+    function strftime_f(format, t) {
+        throw new NotImplementedError("time.strftime() is not yet implemented");
+        Sk.builtin.pyCheckArgs("strftime", arguments, 1, 2);
```

Comment:
I think we should not ship dead code, even when it is the start of an implementation. Better we ship here a clean impl. of the working functions and add the others later.